src/common/image.c

```diff
@@ -744,8 +744,11 @@ int32_t dt_image_duplicate_with_version(const int32_t imgid, const int32_t newve
     sqlite3_finalize(stmt);
 
     // make sure that the duplicate doesn't have some magic darktable| tags
-    dt_tag_detach_by_string("darktable|changed", newid);
```

Comment:
As above, it seems to me that we want to always detach the tags here as it was done.